frontend/rolecall/src/app/cast/cast-editor-v2.component.ts

```diff
@@ -94,10 +102,26 @@ export class CastEditorV2 implements OnInit {
     }
   }
 
+  buildPopupList() {
+    this.popupList = this.allPieces.map((piece, pieceIx) => {
+      const existingCasts = this.allCasts.filter((val) => val.segment == piece.uuid);
+      const prefix = existingCasts.length === 0 ? " " : "z";
```

Comment:
What does the "z" mean? Just for sorting? Feels like it would be better to not take a shortcut by reusing default alphabetical sort, and just add the sort logic inside the sort function below.

Reply:
Yes, the "z" is just for sorting. I disagree that it is a shortcut. This list has alway been sorted alphabetically. The idea here is that every piece that is lacking a cast should be at the top of the list (and the name should preceded with a '*' to show that it hadn't been casted yet. Prepending " " and "z" to the old sort name is very easy to understand and requires a minimal amount of extra code, two goals of good code in my opinion.